api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -232,7 +225,7 @@ public static UserContext getUserContext() {
 		Object[] arr = userContextHolder.get();
 		
 		if (log.isTraceEnabled()) {
-			log.trace("Getting user context " + arr + " from userContextHolder " + userContextHolder);
+			log.trace("Getting user context " + Arrays.toString(arr) + " from userContextHolder " + userContextHolder);
```

Comment:
Is Arrays.toString accepting nulls? arr may be null and we do not want to see NullPointerException so please check that.